### PR TITLE
Update-project-setup

### PR DESCRIPTION
**Pull request by abdallahshaban557**
#### Description of changes:

Update-project-setup based on updates to v6

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [X] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [X] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by nadetastic — DISMISSED** [review]
LGTM

File: src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx [hunk 1]
```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
...
+
+When you initialize a new Amplify project, a few things happen:
+
+- It creates a top level directory called `amplify` that stores your backend
+definition. During the tutorial you'll add capabilities such as a GraphQL API
+and authentication. As you add features, the `amplify` folder will grow with
+infrastructure-as-code templates that define your backend stack.
+Infrastructure-as-code is a best practice way to create a replicable backend stack.
+- It creates a file called `amplifyconfiguration.json` in the `src` directory that holds
+all the configuration for the services you create with Amplify. This is how the
```
Comment (cwomack) [inline src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:248]: nit - should this be starting a new line?

File: src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx [hunk 2]
```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
...
+Amplify client is able to get the necessary information about your backend services.
+- It modifies the `.gitignore` file, adding some generated files to the ignore list
+- A cloud project is created for you in the AWS Amplify Console that can be
+accessed by running `amplify console`. The Console provides a list of backend
+environments, deep links to provisioned resources per Amplify category, status
+of recent deployments, and instructions on how to promote, clone, pull, and delete
+backend resources
```
Comment (cwomack) [inline src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:255]: nit - needs a period.

Comment (cwomack) [inline src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:255]:
```suggestion
backend resources.
```

**Review by cwomack — DISMISSED** [review]
LGTM

**Review by cwomack — DISMISSED** [review]
LGTM!

**Review by nadetastic — APPROVED** [review]

File: src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx [hunk 3]
```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
...
+```
+
+This runs a development server and see preview your app locally.
 
 </Block>
 
 <Block name="VueJS">
 
-<Fragments fragments={{ js: js3 }} />
+Use the Vue Vite powered `create-app` to bootstrap a new Vue 3 app (selecting the defaults will work for this project):
```
Comment (ashwinkumar6) [inline src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:156]:
```suggestion
Use the Vue Vite powered `create-app` to bootstrap a new Vue 3 app (select required configuration, selecting the defaults as an example for this project):
```

Comment (ashwinkumar6) [reply src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:156]: nit, please feel free to ignore if it doesn't make sense

Comment (abdallahshaban557) [reply src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx]: Committed the change!

**Review by cwomack — APPROVED** [review]
Looks good!

**Review by ashwinkumar6 — APPROVED** [review]
Thanks @abdallahshaban557
lgtm

**Review by kvramyasri7 — APPROVED** [review]
All Is Well

File: src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx [hunk 4]
```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
 
 <InlineFilter filters={['js']}>
 
-import js0 from '/src/fragments/lib-v1/start/getting-started/angular/setup.mdx';
```
Comment (renebrandel) [inline src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx:32]: are these fragments used elsewhere? If not, then can you also commit the file deletions? #defrag

**Review by renebrandel — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**